waspc/data/Generator/templates/server/src/core/auth/prismaMiddleware.js

```diff
@@ -2,26 +2,40 @@
 import { hashPassword } from '../auth.js'
 import { PASSWORD_FIELD } from '../../auth/validation.js'
 
+let isPasswordHashingEnabled = true
+
+export async function withDisabledPasswordHashing(action) {
```

Comment:
We can enable migrations from old auth system to the new system with seed scripts.

1. Get all `users`
2. Create new `auth` models
3. Copy over the existing password <---- at this point the password should be saved as-is, not double hashed

Reply:
Hmmm but how are going to do that? So they want to migrate -> how are they going to disable password hashing? Can't we just do SQL queries that will move stuff around correctly?
Maybe you could describe this in more details?

Reply:
> But for that we need to provide a way to disable password hashing temporarily.

This was a leftover from before, sorry about that. The way they can do the migration is to just copy over the hashed password and serialize the `providerData` with `JSON.stringify`. No need to "disable hashing" since no implicit hashing is happening anymore, but it's required to do it explicitly now.

Reply:
Ok and how are they going to do this migration, we are going to give them some code? I guess that is yet to come and I can just wait for code/docs on that?